README.md

```diff
@@ -1,3 +1,23 @@
+##
+Updates to the Readme.md by Dean Smythe
+See diagrams in Excel document and pdf.
+Chitter_Domain_Model.xlsx
+Chitter_Domain_Model.pdf
+
```

Comment:
It's great that you're using doman modelling and it'll become particularly powerful once you're working in a team.